__tests__/integration.test.js

```diff
@@ -71,4 +73,45 @@ describe("GET requests", () => {
         });
     });
   });
+
+  describe("/api/articles/:article_id", () => {
+    it('returns with 200 status code "OK"', () => {
+      return request(app).get("/api/articles/3").expect(200);
+    });
+
+    it("responds with the correct article object containing the required properties", () => {
+      return request(app)
+        .get("/api/articles/1")
+        .then(({ body: { article } }) => {
+          const isObject = Object.keys(article[0]).length > 0;
+          const isNotArray = !Array.isArray(article[0]);
+
+          const properties = [
+            "title",
+            "author",
+            "topic",
+            "article_id",
+            "body",
+            "created_at",
+            "votes",
+            "article_img_url",
+          ];
+
+          expect(isObject && isNotArray).toBe(true);
+
+          for (let i = 0; i < properties.length; i++) {
+            expect(article[0].hasOwnProperty(properties[i])).toBe(true);
+          }
+        });
+    });
+
+    it("returns a 404 if no id is found, inc. a meaningful error", () => {
+      return request(app)
+        .get("/api/articles/600")
+        .expect(404)
+        .then(({ body: { msg } }) => {
+          expect(msg).toBe("ID does not exist");
+        });
+    });
+  });
```

Comment:
Some good tests here Max. What would happen some provided the endpoint  - /api/articles/notvalid - would you treat this the same way? 

The answer should be no...an article with an id of 600 could exist at some point, so sending back a 404 status code is fine as it doesn't exist...YET. 

An article with an id that isn't a number is never going to exist and should be considered a bad request. These are usually sent back on a 400 status code and your server should behave in the same way.